UefiPayloadPkg/BlSupportDxe/BlSupportDxe.inf

```diff
@@ -53,6 +53,7 @@
   gEfiMdeModulePkgTokenSpaceGuid.PcdSetupVideoVerticalResolution
   gEfiMdePkgTokenSpaceGuid.PcdPciExpressBaseAddress
   gEfiMdePkgTokenSpaceGuid.PcdPciExpressBaseSize
+  gUefiPayloadPkgTokenSpaceGuid.PcdPayloadStackTop
 

```

Comment:
The idea way is to remove PcdPayloadStackTop before upstream.
Let's discuss it in the meeting.